features/step_definitions/emails.rb

```diff
@@ -29,3 +29,9 @@ def self.get_email(message_type, email_address)
   @email_address = randomString + '@example.gov.uk'
   puts "Email address: #{@email_address}"
 end
+
+Given /^I have a random email address$/ do
+  randomString = SecureRandom.hex
+  @email_address = randomString + '@example.com'
+  puts "Email address: #{@email_address}"
+end
```

Comment:
An EOF char has sneaked in here...

Reply:
Or rather, a missing line terminator.  Will fix.